source/geod24/concurrency.d

```diff
@@ -982,26 +991,24 @@ private:
         import std.algorithm.mutation : remove;
         while (m_fibers.length > 0)
         {
-            // Is Fiber waiting on a FiberCondition?
-            if (m_fibers[m_pos].cond !is null)
+            // Is Fiber waiting on a FiberBinarySemaphore?
+            if (m_fibers[m_pos].sem !is null)
```

Comment:
As suggested in the previous commit, you can use `if (auto sem = m_fibers[m_pos].sem)` and then use `sem` knowing that it's non-null.